pkg/external_repos/introspect_mocks_test.go

```diff
@@ -28,6 +28,10 @@ type MockRepositoryDao struct {
 	mock.Mock
 }
 
+func (m *MockRepositoryDao) FetchRepositoryRPMCount(repoUUID string) (int, error) {
+	return 13, nil
```

Comment:
There is only one test currently making use of this. So the return value matches that. 
I likely would want to do something similar here if I had more insight.

Reply:
I would do `return args.Int(0), args.Error(1)` and then values can be set in the test itself. For example, in [this test](https://github.com/content-services/content-sources-backend/blob/main/pkg/handler/repositories_test.go#L303) there's `.Return(repo, nil)`. In your case, you might do `.Return(13, nil)`.

Reply:
Cool.